external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/EventDataScheme.java

```diff
@@ -46,25 +42,11 @@
 	private static final long serialVersionUID = 1L;
 
 	@Override
-	public List<Object> deserialize(Message message) {
+	public List<Object> deserialize(EventData eventData) {
 		final List<Object> fieldContents = new ArrayList<Object>();
-
-		Map metaDataMap = new HashMap();
 		String messageData = "";
-
-		for (Section section : message.getPayload()) {
-			if (section instanceof Data) {
-				Data data = (Data) section;
-				messageData = new String(data.getValue().getArray());
-			} else if (section instanceof AmqpValue) {
-				AmqpValue amqpValue = (AmqpValue) section;
-				messageData = amqpValue.getValue().toString();
-			} else if (section instanceof ApplicationProperties) {
-				final ApplicationProperties applicationProperties = (ApplicationProperties) section;
-				metaDataMap = applicationProperties.getValue();
-			}
-		}
-
+		messageData = new String (eventData.getBody(),eventData.getBodyOffset(),eventData.getBodyLength(),Charset.defaultCharset());

```

Comment:
Thanks. Is there any changes for now that you see in the serialization code that needs to be incorporated? Otherwise I will keep an eye for the 0.12.0 version and then incorporate AmqpValue also as a payload